docs/ENVS.md

```diff
@@ -841,6 +841,12 @@ This feature enables Blockscout Merits program. It requires that the [My account
 | --- | --- | --- | --- | --- | --- | --- |
 | NEXT_PUBLIC_REWARDS_SERVICE_API_HOST | `string` | API URL | - | - | `https://example.com` | v1.36.0+ |
 
+### Badge claim link
+
+| Variable | Type| Description | Compulsoriness  | Default value | Example value | Version |
+| --- | --- | --- | --- | --- | --- | --- |
+| NEXT_PUBLIC_BADGE_CLAIM_LINK | `string` | Provide to enable the easter egg badge feature | - | - | `https://example.com` | v1.37.0+ |
+
```

Comment:
Let's make it more descriptive, as the current name does not clearly indicate which badge this link relates to.
For example, `NEXT_PUBLIC_GAME_BADGE_CLAIM_LINK`, or `NEXT_PUBLIC_EASTER_EGG_BADGE_CLAIM_LINK`